x-pack/legacy/plugins/canvas/types/state.ts

```diff
@@ -32,7 +32,6 @@ export interface AppState {
 }
 
 interface StoreAppState {
-  kbnVersion: string;
```

Comment:
I couldn't find any usage of the `kbnVersion` in our frontend code so I'm removing it